tests/performance_tests/performance_test_basic.py

```diff
@@ -159,6 +165,7 @@ def __init__(self, testHelperConfig: TestHelperConfig=TestHelperConfig(), cluste
         self.etcEosioLogsDirPath = self.etcLogsDirPath/Path("eosio")
         self.blockDataLogDirPath = self.loggingConfig.logDirPath/Path("blockDataLogs")
         self.blockDataPath = self.blockDataLogDirPath/Path("blockData.txt")
+        self.csvDataPath = self.blockDataLogDirPath/Path("csv.txt")
```

Comment:
Should this be named more like: `transactionMetrics.csv`?

Reply:
For file names lets use `_` instead of camelCase.